.github/workflows/ci-standard-checks.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   ci-standard-checks:
+    if: ${{ github.repository_owner == 'test' }}
```

Comment:
This will prevent to execute on a fork, but it forces us to modify all workflows in repos. Not that nice 😢